docs/guide/usage/integrations/home_assistant.md

```diff
@@ -39,60 +39,65 @@ The device specific configuration allows you to modify the discovery payload. He
 
 To respond to button clicks (e.g. WXKG01LM) you can use one of the following three Home Assistant configurations.
 
-### Via MQTT device trigger (recommended)
+### Via Home Assistant `event` entity (experimental)
 
-[MQTT device trigger](https://www.home-assistant.io/integrations/device_trigger.mqtt/) is the recommended way to respond to button clicks.
-The MQTT device triggers are discovered by Zigbee2MQTT **once the event is triggered on the device at least once**.
+This method work by responding to the state change of an [`event` entity](https://www.home-assistant.io/integrations/event). The specific event can be targetted via the `event_type` attribute. This will become the recommended method with 2.0.0. Until then, the event types and additional attributes are subject to change and you have to enable `event` entities explicitely by setting `homeassistant: {experimental_event_entities: true}` (see [Configuration](../../configuration/homeassistant.md) for more info).
```

Comment:
```suggestion
Note: `event` entity is **experimental** and will **break** in the future.

This method work by responding to the state change of an [`event` entity](https://www.home-assistant.io/integrations/event). The specific event can be targetted via the `event_type` attribute. This will become the recommended method with 2.0.0. Until then, the event types and additional attributes are subject to change and you have to enable `event` entities explicitely by setting `homeassistant: {experimental_event_entities: true}` (see [Configuration](../../configuration/homeassistant.md) for more info).
```

Reply:
I have added the note (but with "may" instead of "will").